### PR TITLE
Updated patchfoo and patch SSBNpm requirements

### PR DESCRIPTION
**Pull request by darkdrgn2k**

### CODE REVIEW

File: scripts/ssb-patchfoo/install [hunk 1]
```diff
@@ -33,11 +34,13 @@ cp "$BASE_DIR/config" ~/.ssb
 # Stop ssb service to process plugin
 sudo systemctl stop ssb
 
-# Disable the git-ssb prerequisite
-# Comment out two lines in patchwork that create a prerequisite for git-ssb
-# but don't seem to serve any purpose. Git-ssb is not available on npm
+# Disable the git-ssb and npm-ssb prerequisite
+# Comment out two lines in patchwork that create a prerequisite for git-ssb and npm-ssb
+# but don't seem to serve any purpose. git-ssb and and npm-ssbis not available on npm
```
Comment (makeworld-the-better-one) [inline scripts/ssb-patchfoo/install]:
```suggestion
```

Those lines do serve a purpose, it's not like it used to be where they were deps for no reason.

**Comment by makeworld-the-better-one** [comment]
Have you tested this and it works and displays other message types fine? In #398 I showed a lot of lines that require git. Are you sure it will work without the dependency and not cause errors?

**Comment by darkdrgn2k** [comment]
I have not found any, you are welcome to test!

**Comment by makeworld-the-better-one** [comment]
If you know that it works for displaying and posting messages, I'm fine to merge.

**Comment by darkdrgn2k** [comment]
Synced with buster patches since there are related items in there to.

**Pull request merged**